core/tests/puppeteer-acceptance-tests/utilities/user/exploration-editor.ts

```diff
@@ -874,6 +894,10 @@ export class ExplorationEditor extends BaseUser {
         await this.clickOn(addResponseOptionButton);
         await this.type(textInputInteractionOption, answer);
         break;
+      // Add cases for other interaction types here
```

Comment:
What are these comments for?

Reply:
Different Interactions have different types of input field, so in this function there will be different cases for each type of interaction.
Also, this comment was already there. I have just pulled it down after adding a new case above.